Fun Commands/Mini-Games/Dyno-RPG/curse.txt

```diff
@@ -0,0 +1,15 @@
+{delete}
+{silent}
+{require:{&Adventurer}}
+{!role $1 +Cursed}
+**$1** has been cursed...
+
+**$1** attempt to move, but quickly realizes the futility.... 
```

Comment:
Again a little spelling thing (most of these are nitpicky things), might want:
```suggestion
**$1** attempts to move, but quickly realizes the futility.... 
```

Reply:
Fixed